oak/server/wasm_node.cc

```diff
@@ -234,7 +232,8 @@ void WasmNode::Run() {
 
   LOG(INFO) << "module execution thread: run oak_main";
   wabt::interp::TypedValues args;
-  wabt::interp::ExecResult exec_result = executor.RunExportByName(module_, "oak_main", args);
+  wabt::interp::ExecResult exec_result =
+      executor.RunExportByName(env_.GetLastModule(), "oak_main", args);
```

Comment:
How about adding a method `Module()` defined as `env_.GetLastModule()` for clarity?

Reply:
That gets my vote.